proposals/stage-1/CODE_OF_CONDUCT/HANDLING-REPORTS.md

```diff
@@ -0,0 +1,40 @@
+# Handling reports and escalation
+This document describes the proposed process for handling reports and escalation, along with relevant roles in the OpenJS Foundation.
+
+Reporting to the OpenJS Foundation either concerns a matter with the Foundation or is considered an escalation where insufficient resolution was provided by a project, with regards to violations of the Code of Conduct.
+
+## Channels
+* Email: report@openjsf.org.
+* Form: It would be easier on the reportee to have a form template to help them structure their report. The form submission could trigger an email.
+* Repo: openjs/moderation (not provisioned). As commented in PR review, hosting sensitive personal information on GitHub needs due consideration, review, and possibly legal consultation.
+  * A form submission could open an issue in the repo, to allow folks to discuss in context.
+## Confidentiality and record-keeping
+Everything reported and discussed is confidential unless explicitly stated. For any facts to be made public, the reportee’s explicit permission is needed. All reports should be recorded, together with the discussion of it. A private moderation repo would satisfy the need for recordkeeping.
+## Steps/Escalation
+1. A report is filed: This is the first step, mostly defined in Channels above.
+2. The right people are alerted: See Roles.
+3. Contact person assigned: See Roles.
+4. Report is acknowledged: The contact person responds to let the reportee know that the report is being discussed.
+5. Facts and opinions are gathered: Time is allocated to collect information in one place to make sure everyone involved has access.
+6. Facts are discussed: The facts are discussed in context of opinions. Also see **What to look forward to** for guidelines on appropriate response.
+7. An action to be taken is arrived at: The action to be taken is decided by consensus (see section on Consensus process) involving the roles listed in the next section.
+8. Resolution offered: The resolution is offered to the reportee by the contact person. The reportee needs to accept the resolution for the matter to be resolved. If they don’t accept, go back to step 5. If there isn't a clear way to resolve the report, the report can be reviewed by the moderation team representatives from OpenJS projects to ensure that the the action and resolution are appropriate.
+The report is resolved: The report is resolved if:
+* the reportee accepts the action.
+* there is no communication for 21 days on behalf of the original reporter.
+* the Ombudsperson decides that the resolution as it stands is the best possible outcome within the constraints of all involved.
+## Roles
```

Comment:
This is an excellent opportunity to clarify language.

I had used the word "reportee" to refer to the person/s against whom a violation was committed.

I used the word "reporter" when the person who made the report is not the same as the person against whom the violation was committed.

When we clarify, I can go in and change to make the language consistent, it may not be.

We also need a word to refer to the person/s who committed the violation.